src/m365/spo/commands/navigation/navigation-node-set.ts

```diff
@@ -107,10 +107,16 @@ class SpoNavigationNodeSetCommand extends SpoCommand {
       if (this.verbose) {
         logger.logToStderr(`Setting navigation node...`);
       }
+
+      let url = args.options.url;
+      if (!url) {
```

Comment:
Let's do what is defined in the specs, the linkless header is only set when an empty string is provided as URL.